cypress/fixtures/stubbing/default/datasets.js

```diff
@@ -45,7 +46,7 @@ export const DEFAULT_DATASET = {
   creationDate: null,
   refreshDate: null,
   queries: null,
-  security: null,
+  security: { default: ROLES.SCENARIO.ADMIN, accessControlList: [] },
```

Comment:
I'm not sure that setting everyone as admin of the default dataset used in fixtures is the best way to detect RBAC errors in the front-end, we should try to set the lowest level of permissions required for our tests

Reply:
It is useless indeed but I change it to _none_ in dataset sharing.
Anyway, I will rollback security to _null_